### PR TITLE
Add Discord sign in button

### PR DESCRIPTION
**Pull request by code-ga**
#14

### CODE REVIEW

File: lib/create_button.dart [hunk 1]
```diff
@@ -201,6 +217,11 @@ class SignInButton extends StatelessWidget {
         btnTextColor ??= Colors.white;
         btnColor ??= Color(0xFF1877F2);
         break;
+      case ButtonType.discord:
+        btnText ??= "Sign in with Discord";
+        btnTextColor ??= Colors.white;
+        btnColor ??= Color.fromRGBO(88, 101, 241, 1);
```
Comment (emreesen27) [inline lib/create_button.dart:223]: please add color as hex for code compatibility.

**Comment by emreesen27** [comment]
Thanks for your contribution.

**Comment by tobycm** [comment]
nice

**Comment by code-ga** [comment]
=>>

**Comment by tobycm** [comment]
Although, you may get this exception while rendering 
image

**Comment by emreesen27** [comment]
@tobycm yes, there was an issue with the icon size. Fixed.

**Comment by tobycm** [comment]
Nice

**Pull request merged**